src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -282,6 +282,28 @@ def get_top_5_issues_by_count_for_file(
     )
     project_ids = [p.id for p in projects]
 
+    stackframe_function_name = lambda i: Function(
+        "arrayElement",
+        (Column("exception_frames.function"), i),
+    )
+    multi_if = []
+    for i in range(-STACKFRAME_COUNT, 0):
+        # if, then conditions
+        multi_if.extend(
+            [
+                Function(
+                    "in",
+                    [
+                        stackframe_function_name(i),
+                        function_names,
+                    ],
+                ),
+                stackframe_function_name(i),
+            ]
+        )
+    # else condition
+    multi_if.append(stackframe_function_name(-1))
```

Comment:
Does the iterator not add the last element of the stackframe_function_name list?

Reply:
this is the ELSE condition, the syntax is `multiIf(cond_1, then_1, cond_2, then_2, ..., else)`
https://clickhouse.com/docs/en/sql-reference/functions/conditional-functions#multiif

Reply:
we shouldn't hit this but i think we need it